plantisc2/tasks/geerlingguy.mysql/defaults/main.yml

```diff
@@ -0,0 +1,121 @@
+---
+# Set this to the user ansible is logging in as - should have root
+# or sudo access
+mysql_user_home: /root
+mysql_user_name: root
+mysql_user_password: root
```

Comment:
Keep passwords protected in a vault file.

Ensure that you have an `- include_vars: ` task to pull in the variables. do not keep passwords in defaults.